src/workerd/io/actor-cache-test.c++

```diff
@@ -3700,7 +3700,8 @@ KJ_TEST("ActorCache evict on timeout") {
 }
 
 KJ_TEST("ActorCache backpressure due to dirtyPressureThreshold") {
-  ActorCacheTest test({.dirtyKeySoftLimit = 2});
+  // Each Entry below ends up being about ~126 bytes, so a limit of 256 allows for 2 entries.
```

Comment:
no action: 126 bytes huh? I'd guess that some of that is v8 encoding and the general use of maybes. I bet we could trim down at least a few bytes by doing some struct layout optimization.

Reply:
Yeah, it surprised me too. An effectively empty `Entry` carries around quite a few bytes.

Reply:
v8 encoding doesn't add nearly that much by itself on a simple value -- it should just be 2-3 bytes of overhead, e.g.:

```
$ node
Welcome to Node.js v18.6.0.
> v8.serialize(true).length
3
> v8.serialize(null).length
3
> v8.serialize(0).length
4
```

Reply:
Actually looking how many different fields are involved an `Entry`, it's not a surprise that there's lots of overhead: https://github.com/cloudflare/workerd/blob/38f79147e35e9c265d6da6756bac70714282c38b/src/workerd/io/actor-cache.h#L336-L413